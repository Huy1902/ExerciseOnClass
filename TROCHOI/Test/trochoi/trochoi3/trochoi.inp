200
aaaaa1234kfjlgdklgdfk
bbbbbbb12kfjgldfgldfgjldfgld
ccccc1fkgggggggjjgjjjjjjjjjfgdfg
ddddddd234fggggggg
eeeeeeeeee12345bbbbbbbbbbbbbbc
aaaaaaaaaa123456fjjjjjjjjjjjjjj
aaaaaaaaaaaa123456789gffffffffffff
fddddddddddddd1234567jhhhhhhhhh
aaaaaaaaaaaa12345678ghhhhh
ssssss1234567891fdddddddddddfddddddf
aaaaa1234kfjlgdklgdfk
bbbbbbb12kfjgldfgldfgjldfgld
ccccc1fkgggggggjjgjjjjjjjjjfgdfg
ddddddd234fggggggg
eeeeeeeeee12345bbbbbbbbbbbbbbc
aaaaaaaaaa123456fjjjjjjjjjjjjjj
aaaaaaaaaaaa123456789gffffffffffff
fddddddddddddd1234567jhhhhhhhhh
aaaaaaaaaaaa12345678ghhhhh
ssssss1234567891fdddddddddddfddddddf
aaaaa1234kfjlgdklgdfk
bbbbbbb12kfjgldfgldfgjldfgld
ccccc1fkgggggggjjgjjjjjjjjjfgdfg
ddddddd234fggggggg
eeeeeeeeee12345bbbbbbbbbbbbbbc
aaaaaaaaaa123456fjjjjjjjjjjjjjj
aaaaaaaaaaaa123456789gffffffffffff
fddddddddddddd1234567jhhhhhhhhh
aaaaaaaaaaaa12345678ghhhhh
ssssss1234567891fdddddddddddfddddddf
aaaaa1234kfjlgdklgdfk
bbbbbbb12kfjgldfgldfgjldfgld
ccccc1fkgggggggjjgjjjjjjjjjfgdfg
ddddddd234fggggggg
eeeeeeeeee12345bbbbbbbbbbbbbbc
aaaaaaaaaa123456fjjjjjjjjjjjjjj
aaaaaaaaaaaa123456789gffffffffffff
fddddddddddddd1234567jhhhhhhhhh
aaaaaaaaaaaa12345678ghhhhh
ssssss1234567891fdddddddddddfddddddf
aaaaa1234kfjlgdklgdfk
bbbbbbb12kfjgldfgldfgjldfgld
ccccc1fkgggggggjjgjjjjjjjjjfgdfg
ddddddd234fggggggg
eeeeeeeeee12345bbbbbbbbbbbbbbc
aaaaaaaaaa123456fjjjjjjjjjjjjjj
aaaaaaaaaaaa123456789gffffffffffff
fddddddddddddd1234567jhhhhhhhhh
aaaaaaaaaaaa12345678ghhhhh
ssssss1234567891fdddddddddddfddddddf
aaaaa1234kfjlgdklgdfk
bbbbbbb12kfjgldfgldfgjldfgld
ccccc1fkgggggggjjgjjjjjjjjjfgdfg
ddddddd234fggggggg
eeeeeeeeee12345bbbbbbbbbbbbbbc
aaaaaaaaaa123456fjjjjjjjjjjjjjj
aaaaaaaaaaaa123456789gffffffffffff
fddddddddddddd1234567jhhhhhhhhh
aaaaaaaaaaaa12345678ghhhhh
ssssss1234567891fdddddddddddfddddddf
aaaaa1234kfjlgdklgdfk
bbbbbbb12kfjgldfgldfgjldfgld
ccccc1fkgggggggjjgjjjjjjjjjfgdfg
ddddddd234fggggggg
eeeeeeeeee12345bbbbbbbbbbbbbbc
aaaaaaaaaa123456fjjjjjjjjjjjjjj
aaaaaaaaaaaa123456789gffffffffffff
fddddddddddddd1234567jhhhhhhhhh
aaaaaaaaaaaa12345678ghhhhh
ssssss1234567891fdddddddddddfddddddf
aaaaa1234kfjlgdklgdfk
bbbbbbb12kfjgldfgldfgjldfgld
ccccc1fkgggggggjjgjjjjjjjjjfgdfg
ddddddd234fggggggg
eeeeeeeeee12345bbbbbbbbbbbbbbc
aaaaaaaaaa123456fjjjjjjjjjjjjjj
aaaaaaaaaaaa123456789gffffffffffff
fddddddddddddd1234567jhhhhhhhhh
aaaaaaaaaaaa12345678ghhhhh
ssssss1234567891fdddddddddddfddddddf
aaaaa1234kfjlgdklgdfk
bbbbbbb12kfjgldfgldfgjldfgld
ccccc1fkgggggggjjgjjjjjjjjjfgdfg
ddddddd234fggggggg
eeeeeeeeee12345bbbbbbbbbbbbbbc
aaaaaaaaaa123456fjjjjjjjjjjjjjj
aaaaaaaaaaaa123456789gffffffffffff
fddddddddddddd1234567jhhhhhhhhh
aaaaaaaaaaaa12345678ghhhhh
ssssss1234567891fdddddddddddfddddddf
aaaaa1234kfjlgdklgdfk
bbbbbbb12kfjgldfgldfgjldfgld
ccccc1fkgggggggjjgjjjjjjjjjfgdfg
ddddddd234fggggggg
eeeeeeeeee12345bbbbbbbbbbbbbbc
aaaaaaaaaa123456fjjjjjjjjjjjjjj
aaaaaaaaaaaa123456789gffffffffffff
fddddddddddddd1234567jhhhhhhhhh
aaaaaaaaaaaa12345678ghhhhh
ssssss1234567891fdddddddddddfddddddf
aaaaa1234kfjlgdklgdfk
bbbbbbb12kfjgldfgldfgjldfgld
ccccc1fkgggggggjjgjjjjjjjjjfgdfg
ddddddd234fggggggg
eeeeeeeeee12345bbbbbbbbbbbbbbc
aaaaaaaaaa123456fjjjjjjjjjjjjjj
aaaaaaaaaaaa123456789gffffffffffff
fddddddddddddd1234567jhhhhhhhhh
aaaaaaaaaaaa12345678ghhhhh
ssssss1234567891fdddddddddddfddddddf
aaaaa1234kfjlgdklgdfk
bbbbbbb12kfjgldfgldfgjldfgld
ccccc1fkgggggggjjgjjjjjjjjjfgdfg
ddddddd234fggggggg
eeeeeeeeee12345bbbbbbbbbbbbbbc
aaaaaaaaaa123456fjjjjjjjjjjjjjj
aaaaaaaaaaaa123456789gffffffffffff
fddddddddddddd1234567jhhhhhhhhh
aaaaaaaaaaaa12345678ghhhhh
ssssss1234567891fdddddddddddfddddddf
aaaaa1234kfjlgdklgdfk
bbbbbbb12kfjgldfgldfgjldfgld
ccccc1fkgggggggjjgjjjjjjjjjfgdfg
ddddddd234fggggggg
eeeeeeeeee12345bbbbbbbbbbbbbbc
aaaaaaaaaa123456fjjjjjjjjjjjjjj
aaaaaaaaaaaa123456789gffffffffffff
fddddddddddddd1234567jhhhhhhhhh
aaaaaaaaaaaa12345678ghhhhh
ssssss1234567891fdddddddddddfddddddf
aaaaa1234kfjlgdklgdfk
bbbbbbb12kfjgldfgldfgjldfgld
ccccc1fkgggggggjjgjjjjjjjjjfgdfg
ddddddd234fggggggg
eeeeeeeeee12345bbbbbbbbbbbbbbc
aaaaaaaaaa123456fjjjjjjjjjjjjjj
aaaaaaaaaaaa123456789gffffffffffff
fddddddddddddd1234567jhhhhhhhhh
aaaaaaaaaaaa12345678ghhhhh
ssssss1234567891fdddddddddddfddddddf
aaaaa1234kfjlgdklgdfk
bbbbbbb12kfjgldfgldfgjldfgld
ccccc1fkgggggggjjgjjjjjjjjjfgdfg
ddddddd234fggggggg
eeeeeeeeee12345bbbbbbbbbbbbbbc
aaaaaaaaaa123456fjjjjjjjjjjjjjj
aaaaaaaaaaaa123456789gffffffffffff
fddddddddddddd1234567jhhhhhhhhh
aaaaaaaaaaaa12345678ghhhhh
ssssss1234567891fdddddddddddfddddddf
aaaaa1234kfjlgdklgdfk
bbbbbbb12kfjgldfgldfgjldfgld
ccccc1fkgggggggjjgjjjjjjjjjfgdfg
ddddddd234fggggggg
eeeeeeeeee12345bbbbbbbbbbbbbbc
aaaaaaaaaa123456fjjjjjjjjjjjjjj
aaaaaaaaaaaa123456789gffffffffffff
fddddddddddddd1234567jhhhhhhhhh
aaaaaaaaaaaa12345678ghhhhh
ssssss1234567891fdddddddddddfddddddf
aaaaa1234kfjlgdklgdfk
bbbbbbb12kfjgldfgldfgjldfgld
ccccc1fkgggggggjjgjjjjjjjjjfgdfg
ddddddd234fggggggg
eeeeeeeeee12345bbbbbbbbbbbbbbc
aaaaaaaaaa123456fjjjjjjjjjjjjjj
aaaaaaaaaaaa123456789gffffffffffff
fddddddddddddd1234567jhhhhhhhhh
aaaaaaaaaaaa12345678ghhhhh
ssssss1234567891fdddddddddddfddddddf
aaaaa1234kfjlgdklgdfk
bbbbbbb12kfjgldfgldfgjldfgld
ccccc1fkgggggggjjgjjjjjjjjjfgdfg
ddddddd234fggggggg
eeeeeeeeee12345bbbbbbbbbbbbbbc
aaaaaaaaaa123456fjjjjjjjjjjjjjj
aaaaaaaaaaaa123456789gffffffffffff
fddddddddddddd1234567jhhhhhhhhh
aaaaaaaaaaaa12345678ghhhhh
ssssss1234567891fdddddddddddfddddddf
aaaaa1234kfjlgdklgdfk
bbbbbbb12kfjgldfgldfgjldfgld
ccccc1fkgggggggjjgjjjjjjjjjfgdfg
ddddddd234fggggggg
eeeeeeeeee12345bbbbbbbbbbbbbbc
aaaaaaaaaa123456fjjjjjjjjjjjjjj
aaaaaaaaaaaa123456789gffffffffffff
fddddddddddddd1234567jhhhhhhhhh
aaaaaaaaaaaa12345678ghhhhh
ssssss1234567891fdddddddddddfddddddf
aaaaa1234kfjlgdklgdfk
bbbbbbb12kfjgldfgldfgjldfgld
ccccc1fkgggggggjjgjjjjjjjjjfgdfg
ddddddd234fggggggg
eeeeeeeeee12345bbbbbbbbbbbbbbc
aaaaaaaaaa123456fjjjjjjjjjjjjjj
aaaaaaaaaaaa123456789gffffffffffff
fddddddddddddd1234567jhhhhhhhhh
aaaaaaaaaaaa12345678ghhhhh
ssssss1234567891fdddddddddddfddddddf
