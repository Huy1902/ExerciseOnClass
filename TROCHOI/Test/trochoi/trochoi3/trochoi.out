ccccc1fkgggggggjjgjjjjjjjjjfgdfg
ccccc1fkgggggggjjgjjjjjjjjjfgdfg
ccccc1fkgggggggjjgjjjjjjjjjfgdfg
ccccc1fkgggggggjjgjjjjjjjjjfgdfg
ccccc1fkgggggggjjgjjjjjjjjjfgdfg
ccccc1fkgggggggjjgjjjjjjjjjfgdfg
ccccc1fkgggggggjjgjjjjjjjjjfgdfg
ccccc1fkgggggggjjgjjjjjjjjjfgdfg
ccccc1fkgggggggjjgjjjjjjjjjfgdfg
ccccc1fkgggggggjjgjjjjjjjjjfgdfg
ccccc1fkgggggggjjgjjjjjjjjjfgdfg
ccccc1fkgggggggjjgjjjjjjjjjfgdfg
ccccc1fkgggggggjjgjjjjjjjjjfgdfg
ccccc1fkgggggggjjgjjjjjjjjjfgdfg
ccccc1fkgggggggjjgjjjjjjjjjfgdfg
ccccc1fkgggggggjjgjjjjjjjjjfgdfg
ccccc1fkgggggggjjgjjjjjjjjjfgdfg
ccccc1fkgggggggjjgjjjjjjjjjfgdfg
ccccc1fkgggggggjjgjjjjjjjjjfgdfg
ccccc1fkgggggggjjgjjjjjjjjjfgdfg
bbbbbbb12kfjgldfgldfgjldfgld
bbbbbbb12kfjgldfgldfgjldfgld
bbbbbbb12kfjgldfgldfgjldfgld
bbbbbbb12kfjgldfgldfgjldfgld
bbbbbbb12kfjgldfgldfgjldfgld
bbbbbbb12kfjgldfgldfgjldfgld
bbbbbbb12kfjgldfgldfgjldfgld
bbbbbbb12kfjgldfgldfgjldfgld
bbbbbbb12kfjgldfgldfgjldfgld
bbbbbbb12kfjgldfgldfgjldfgld
bbbbbbb12kfjgldfgldfgjldfgld
bbbbbbb12kfjgldfgldfgjldfgld
bbbbbbb12kfjgldfgldfgjldfgld
bbbbbbb12kfjgldfgldfgjldfgld
bbbbbbb12kfjgldfgldfgjldfgld
bbbbbbb12kfjgldfgldfgjldfgld
bbbbbbb12kfjgldfgldfgjldfgld
bbbbbbb12kfjgldfgldfgjldfgld
bbbbbbb12kfjgldfgldfgjldfgld
bbbbbbb12kfjgldfgldfgjldfgld
ddddddd234fggggggg
ddddddd234fggggggg
ddddddd234fggggggg
ddddddd234fggggggg
ddddddd234fggggggg
ddddddd234fggggggg
ddddddd234fggggggg
ddddddd234fggggggg
ddddddd234fggggggg
ddddddd234fggggggg
ddddddd234fggggggg
ddddddd234fggggggg
ddddddd234fggggggg
ddddddd234fggggggg
ddddddd234fggggggg
ddddddd234fggggggg
ddddddd234fggggggg
ddddddd234fggggggg
ddddddd234fggggggg
ddddddd234fggggggg
aaaaa1234kfjlgdklgdfk
aaaaa1234kfjlgdklgdfk
aaaaa1234kfjlgdklgdfk
aaaaa1234kfjlgdklgdfk
aaaaa1234kfjlgdklgdfk
aaaaa1234kfjlgdklgdfk
aaaaa1234kfjlgdklgdfk
aaaaa1234kfjlgdklgdfk
aaaaa1234kfjlgdklgdfk
aaaaa1234kfjlgdklgdfk
aaaaa1234kfjlgdklgdfk
aaaaa1234kfjlgdklgdfk
aaaaa1234kfjlgdklgdfk
aaaaa1234kfjlgdklgdfk
aaaaa1234kfjlgdklgdfk
aaaaa1234kfjlgdklgdfk
aaaaa1234kfjlgdklgdfk
aaaaa1234kfjlgdklgdfk
aaaaa1234kfjlgdklgdfk
aaaaa1234kfjlgdklgdfk
eeeeeeeeee12345bbbbbbbbbbbbbbc
eeeeeeeeee12345bbbbbbbbbbbbbbc
eeeeeeeeee12345bbbbbbbbbbbbbbc
eeeeeeeeee12345bbbbbbbbbbbbbbc
eeeeeeeeee12345bbbbbbbbbbbbbbc
eeeeeeeeee12345bbbbbbbbbbbbbbc
eeeeeeeeee12345bbbbbbbbbbbbbbc
eeeeeeeeee12345bbbbbbbbbbbbbbc
eeeeeeeeee12345bbbbbbbbbbbbbbc
eeeeeeeeee12345bbbbbbbbbbbbbbc
eeeeeeeeee12345bbbbbbbbbbbbbbc
eeeeeeeeee12345bbbbbbbbbbbbbbc
eeeeeeeeee12345bbbbbbbbbbbbbbc
eeeeeeeeee12345bbbbbbbbbbbbbbc
eeeeeeeeee12345bbbbbbbbbbbbbbc
eeeeeeeeee12345bbbbbbbbbbbbbbc
eeeeeeeeee12345bbbbbbbbbbbbbbc
eeeeeeeeee12345bbbbbbbbbbbbbbc
eeeeeeeeee12345bbbbbbbbbbbbbbc
eeeeeeeeee12345bbbbbbbbbbbbbbc
aaaaaaaaaa123456fjjjjjjjjjjjjjj
aaaaaaaaaa123456fjjjjjjjjjjjjjj
aaaaaaaaaa123456fjjjjjjjjjjjjjj
aaaaaaaaaa123456fjjjjjjjjjjjjjj
aaaaaaaaaa123456fjjjjjjjjjjjjjj
aaaaaaaaaa123456fjjjjjjjjjjjjjj
aaaaaaaaaa123456fjjjjjjjjjjjjjj
aaaaaaaaaa123456fjjjjjjjjjjjjjj
aaaaaaaaaa123456fjjjjjjjjjjjjjj
aaaaaaaaaa123456fjjjjjjjjjjjjjj
aaaaaaaaaa123456fjjjjjjjjjjjjjj
aaaaaaaaaa123456fjjjjjjjjjjjjjj
aaaaaaaaaa123456fjjjjjjjjjjjjjj
aaaaaaaaaa123456fjjjjjjjjjjjjjj
aaaaaaaaaa123456fjjjjjjjjjjjjjj
aaaaaaaaaa123456fjjjjjjjjjjjjjj
aaaaaaaaaa123456fjjjjjjjjjjjjjj
aaaaaaaaaa123456fjjjjjjjjjjjjjj
aaaaaaaaaa123456fjjjjjjjjjjjjjj
aaaaaaaaaa123456fjjjjjjjjjjjjjj
fddddddddddddd1234567jhhhhhhhhh
fddddddddddddd1234567jhhhhhhhhh
fddddddddddddd1234567jhhhhhhhhh
fddddddddddddd1234567jhhhhhhhhh
fddddddddddddd1234567jhhhhhhhhh
fddddddddddddd1234567jhhhhhhhhh
fddddddddddddd1234567jhhhhhhhhh
fddddddddddddd1234567jhhhhhhhhh
fddddddddddddd1234567jhhhhhhhhh
fddddddddddddd1234567jhhhhhhhhh
fddddddddddddd1234567jhhhhhhhhh
fddddddddddddd1234567jhhhhhhhhh
fddddddddddddd1234567jhhhhhhhhh
fddddddddddddd1234567jhhhhhhhhh
fddddddddddddd1234567jhhhhhhhhh
fddddddddddddd1234567jhhhhhhhhh
fddddddddddddd1234567jhhhhhhhhh
fddddddddddddd1234567jhhhhhhhhh
fddddddddddddd1234567jhhhhhhhhh
fddddddddddddd1234567jhhhhhhhhh
aaaaaaaaaaaa12345678ghhhhh
aaaaaaaaaaaa12345678ghhhhh
aaaaaaaaaaaa12345678ghhhhh
aaaaaaaaaaaa12345678ghhhhh
aaaaaaaaaaaa12345678ghhhhh
aaaaaaaaaaaa12345678ghhhhh
aaaaaaaaaaaa12345678ghhhhh
aaaaaaaaaaaa12345678ghhhhh
aaaaaaaaaaaa12345678ghhhhh
aaaaaaaaaaaa12345678ghhhhh
aaaaaaaaaaaa12345678ghhhhh
aaaaaaaaaaaa12345678ghhhhh
aaaaaaaaaaaa12345678ghhhhh
aaaaaaaaaaaa12345678ghhhhh
aaaaaaaaaaaa12345678ghhhhh
aaaaaaaaaaaa12345678ghhhhh
aaaaaaaaaaaa12345678ghhhhh
aaaaaaaaaaaa12345678ghhhhh
aaaaaaaaaaaa12345678ghhhhh
aaaaaaaaaaaa12345678ghhhhh
aaaaaaaaaaaa123456789gffffffffffff
aaaaaaaaaaaa123456789gffffffffffff
aaaaaaaaaaaa123456789gffffffffffff
aaaaaaaaaaaa123456789gffffffffffff
aaaaaaaaaaaa123456789gffffffffffff
aaaaaaaaaaaa123456789gffffffffffff
aaaaaaaaaaaa123456789gffffffffffff
aaaaaaaaaaaa123456789gffffffffffff
aaaaaaaaaaaa123456789gffffffffffff
aaaaaaaaaaaa123456789gffffffffffff
aaaaaaaaaaaa123456789gffffffffffff
aaaaaaaaaaaa123456789gffffffffffff
aaaaaaaaaaaa123456789gffffffffffff
aaaaaaaaaaaa123456789gffffffffffff
aaaaaaaaaaaa123456789gffffffffffff
aaaaaaaaaaaa123456789gffffffffffff
aaaaaaaaaaaa123456789gffffffffffff
aaaaaaaaaaaa123456789gffffffffffff
aaaaaaaaaaaa123456789gffffffffffff
aaaaaaaaaaaa123456789gffffffffffff
ssssss1234567891fdddddddddddfddddddf
ssssss1234567891fdddddddddddfddddddf
ssssss1234567891fdddddddddddfddddddf
ssssss1234567891fdddddddddddfddddddf
ssssss1234567891fdddddddddddfddddddf
ssssss1234567891fdddddddddddfddddddf
ssssss1234567891fdddddddddddfddddddf
ssssss1234567891fdddddddddddfddddddf
ssssss1234567891fdddddddddddfddddddf
ssssss1234567891fdddddddddddfddddddf
ssssss1234567891fdddddddddddfddddddf
ssssss1234567891fdddddddddddfddddddf
ssssss1234567891fdddddddddddfddddddf
ssssss1234567891fdddddddddddfddddddf
ssssss1234567891fdddddddddddfddddddf
ssssss1234567891fdddddddddddfddddddf
ssssss1234567891fdddddddddddfddddddf
ssssss1234567891fdddddddddddfddddddf
ssssss1234567891fdddddddddddfddddddf
ssssss1234567891fdddddddddddfddddddf
